### PR TITLE
[Merged by Bors] - feat: sync Data.Set.Intervals.OrdConnected

### PR DESCRIPTION
**Pull request by joneugster**
---

* [`data.set.intervals.ord_connected`@`9003f28797c0664a49e4179487267c494477d853`..`b19481deb571022990f1baa9cbf9172e6757a479`](https://leanprover-community.github.io/mathlib-port-status/file/data/set/intervals/ord_connected?range=9003f28797c0664a49e4179487267c494477d853..b19481deb571022990f1baa9cbf9172e6757a479)

This matches the way it is in mathlib3, but is it correct that the new statement is `Set.IsAntichain.ordConnected` instead of `IsAntichain.ordConnected`?

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by eric-wieser** [comment]
> This matches the way it is in mathlib3, but is it correct that the new statement is Set.IsAntichain.ordConnected instead of IsAntichain.ordConnected?

no, mathport doesn't know about `_root_`.

**Comment by YaelDillies** [comment]
Uh oh. Let me fix that quick.

File: Mathlib/Data/Set/Intervals/OrdConnected.lean [hunk 1]
```diff
@@ -226,6 +227,19 @@ theorem dual_ordConnected {s : Set α} [OrdConnected s] : OrdConnected (ofDual 
...
+section PartialOrder
+
+variable {α : Type _} [PartialOrder α] {s : Set α}
+
+protected theorem IsAntichain.ordConnected (hs : IsAntichain (· ≤ ·) s) : s.OrdConnected :=
+  ⟨fun x hx y hy z hz => by
+    obtain rfl := hs.eq hx hy (hz.1.trans hz.2)
+    rw [Icc_self, mem_singleton_iff] at hz
+    rwa [hz]⟩
+#align set.is_antichain.ord_connected Set.IsAntichain.ordConnected
```
Comment (YaelDillies) [inline Mathlib/Data/Set/Intervals/OrdConnected.lean:239]:
```suggestion
protected theorem _root_.IsAntichain.ordConnected (hs : IsAntichain (· ≤ ·) s) : s.OrdConnected :=
  ⟨fun x hx y hy z hz => by
    obtain rfl := hs.eq hx hy (hz.1.trans hz.2)
    rw [Icc_self, mem_singleton_iff] at hz
    rwa [hz]⟩
#align is_antichain.ord_connected IsAntichain.ordConnected
```

**Comment by YaelDillies** [comment]
maintainer merge

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**